app/views/users/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<h1 class="text-center">Users</h1>
+
+<div class="paginate" mb-3>
+    <%= will_paginate %>
+</div>
+
+<div class="row">
+    <% @users.each do |user| %>
+        <div class="col-md-6 mb-2">
+            <div class="users_image">
+                <ul>
+                    <% if user.image? %>
+                        <li><%= image_tag(user.image.url) %></li>
+                    <% else %>
+                        <li><%= image_tag("default.png") %></li>
+                    <% end %>
+                    <%= link_to user.name,user_path(user.id) %>
```

Comment:
@hashimoto97 
User `media object` element in bootstrap to display image and user name